easybuild/easyconfigs/e/ecCodes/ecCodes-2.17.0-gompi-2019b.eb

```diff
@@ -2,36 +2,41 @@ easyblock = 'CMakeMake'
 
 name = 'ecCodes'
 version = '2.17.0'
-versionsuffix = '-Python-3.7.4'
 
 homepage = 'https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home'
 description = """ecCodes is a package developed by ECMWF which provides an application programming interface and
  a set of tools for decoding and encoding messages in the following formats: WMO FM-92 GRIB edition 1 and edition 2,
  WMO FM-94 BUFR edition 3 and edition 4, WMO GTS abbreviated header (only decoding)."""
 
-toolchain = {'name': 'foss', 'version': '2019b'}
+toolchain = {'name': 'gompi', 'version': '2019b'}
```

Comment:
@lexming ecCodes seems to use FFTW, so maybe better to keep at foss level?

Reply:
where do you see that FFTW is needed? The only thing I found is a `FindFFTW.cmake` file in the source, but it's not used.

Reply:
indeed, you're right.